spyder/plugins/editor.py

```diff
@@ -2643,3 +2641,23 @@ def set_create_new_file_if_empty(self, value):
         """Change the value of create_new_file_if_empty"""
         for editorstack in self.editorstacks:
             editorstack.create_new_file_if_empty = value
+
+    def move_editorstack_data(self, start, end):
+        """
+        Move editorstack.data to be synchronized when tabs are moved.
+        """
+        if start < 0 or end < 0:
+            return
+        else:
+            steps = abs(end - start)
+            sign = (end - start) // steps

```

Comment:
maybe sign is not the best name hmmm

`direction` or `step` maybe? and adding a comment saying +1 for right, -1 for left?